ios/RNSScreenStackHeaderConfig.m

```diff
@@ -383,6 +388,8 @@ + (void)updateViewController:(UIViewController *)vc withConfig:(RNSScreenStackHe
     }
     navitem.largeTitleDisplayMode = config.largeTitle ? UINavigationItemLargeTitleDisplayModeAlways : UINavigationItemLargeTitleDisplayModeNever;
   }
+#endif
+
 #ifdef __IPHONE_13_0
```

Comment:
Why there is no `#if defined(__IPHONE_13_0) && TARGET_OS_IOS` but it is like that in line 537?

Reply:
This is likely an oversight, changed all other occurences to this.